students/BrianNisonger/Lesson5/assignment/test_database.py

```diff
@@ -0,0 +1,177 @@
+import os
+import pytest
+
+from database import import_data
+from database import show_available_products
+from database import show_rentals
+from database import calculate_availability
+from database import make_product_dict
+from database import return_user_ids
+from database import make_customer_dict
+
+from pymongo import MongoClient
+
+
+class MongoDBConnection(object):
+    """MongoDB Connection"""
+
+    def __init__(self, host='127.0.0.1', port=27017):
+        """ be sure to use the ip address not name for local windows"""
+        self.host = host
+        self.port = port
+        self.connection = None
+
+    def __enter__(self):
+        self.connection = MongoClient(self.host, self.port)
+        return self
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        self.connection.close()
```

Comment:
This looks the same as your mongo connection class in your other file. Just import that one, you don't have to create it again.